frontend/express/public/stylesheets/styles/blocks/_sidebar.scss

```diff
@@ -83,7 +92,6 @@ $bg-color: #24292e;
         padding-bottom: 50px;
         box-sizing: border-box;
         height: 100%;
-        overflow: hidden;
```

Comment:
i think lets keep it hidden.
the outer menu should not be scrollable since it also contains the app selector.